elixir/apps/web/lib/web/controllers/email_otp_controller.ex

```diff
@@ -4,28 +4,12 @@ defmodule Web.EmailOTPController do
   """
   use Web, :controller
 
-  alias Domain.{
-    Actor,
-    Auth,
-    EmailOTP,
-    Token
-  }
-
+  alias Domain.{Auth, EmailOTP}
```

Comment:
If we're going to move to use [Credo.Check.Readability.MultiAlias](https://hexdocs.pm/credo/Credo.Check.Readability.MultiAlias.html) should we go ahead and separate these aliases now?